parsec/core/types/backend_address.py

```diff
@@ -148,9 +157,13 @@ def organization_id(self) -> OrganizationID:
     def bootstrap_token(self) -> str:
         return self._bootstrap_token
 
-    def generate_organization_addr(self, root_verify_key: VerifyKey) -> BackendOrganizationAddr:
+    def generate_organization_addr(
+        self, root_verify_key: VerifyKey, expiration_date: Pendulum = None
+    ) -> BackendOrganizationAddr:
```

Comment:
This change only affects the tests and not the CLI. Anyway, I don't think the expiration date should be used during the bootstrap phase (I would expect to see it appear only in the creation phase).

Reply:
J'imagine qu'à terme on devrait pouvoir modifier cette date d'expiration via une api dédiée (typiquement pour bloquer une organization appartenant à un utilisateur ayant arrêté de payer le service)
Du coup cette information d'expiration peut changer à tout moment et doit être récupérer via une api genre `organization_status`, ce qui rend caduc le fait de fournir cette info dans l'url de l'organization

Reply:
ok